hc/accounts/tests/test_login.py

```diff
@@ -17,15 +17,15 @@ def test_it_sends_link(self):
         r = self.client.post("/accounts/login/", form)
         assert r.status_code == 302
 
-        # ## **Assert that a user was created
+        # Assert that a user was created #
```

Comment:
The diff is sufficient to show the introduced code, which is the base criterion as to whether the task is done.